x-pack/plugins/lists/common/schemas/types/default_comments_array.ts

```diff
@@ -26,24 +21,8 @@ export const DefaultCommentsArray: DefaultCommentsArrayC = new t.Type<
   unknown
 >(
   'DefaultCommentsArray',
-  t.array(comment).is,
-  (input, context): Either<t.Errors, CommentsArray> =>
-    input == null ? t.success([]) : t.array(comment).validate(input, context),
-  t.identity
-);
-
-/**
- * Types the DefaultCommentsPartialArray as:
- *   - If null or undefined, then a default array of type entry will be set
- */
-export const DefaultCommentsPartialArray: DefaultCommentsPartialArrayC = new t.Type<
-  CommentsPartialArray,
-  CommentsPartialArray,
-  unknown
->(
-  'DefaultCommentsPartialArray',
-  t.array(commentPartial).is,
-  (input, context): Either<t.Errors, CommentsPartialArray> =>
-    input == null ? t.success([]) : t.array(commentPartial).validate(input, context),
+  t.array(comments).is,
```

Comment:
NOTE: Previously had the different comment schemas all in one doc, but started to feel really messy. Broke out into their own docs, tests, and mocks.